src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -162,51 +162,27 @@ class SingleActiveFIMap extends React.Component<
   }
 
   public async componentDidMount() {
-    const {
-      fetchGoalsActionCreator,
-      fetchJurisdictionsActionCreator,
-      fetchPlansActionCreator,
-      fetchStructuresActionCreator,
-      fetchTasksActionCreator,
-      plan,
-      supersetService,
-    } = this.props;
-
-    if (plan && plan.plan_id) {
-      /** define superset filter params for jurisdictions */
-      const jurisdictionsParams = superset.getFormData(SUPERSET_MAX_RECORDS, [
-        { comparator: plan.jurisdiction_id, operator: '==', subject: 'jurisdiction_id' },
+    if (!this.props.plan) {
+      /**
+       * Fetch plans again because on reloading the page, the state does not persist
+       * the plans
+       */
+      const { supersetService, fetchPlansActionCreator } = this.props;
+      const supersetParams = superset.getFormData(2000, [
+        { comparator: InterventionType.FI, operator: '==', subject: 'plan_intervention_type' },
       ]);
-      await supersetService(SUPERSET_JURISDICTIONS_SLICE, jurisdictionsParams).then(
-        (result: Jurisdiction[]) => fetchJurisdictionsActionCreator(result)
-      );
-      /** define superset params for filtering by plan_id */
-      const supersetParams = superset.getFormData(SUPERSET_MAX_RECORDS, [
-        { comparator: plan.plan_id, operator: '==', subject: 'plan_id' },
-      ]);
-      /** define superset params for goals */
-      const goalsParams = superset.getFormData(
-        SUPERSET_MAX_RECORDS,
-        [{ comparator: plan.plan_id, operator: '==', subject: 'plan_id' }],
-        { action_prefix: true }
-      );
-      /** Implement Ad hoc Queries since jurisdictions have no plan_id */
-      await supersetService(SUPERSET_STRUCTURES_SLICE, jurisdictionsParams).then(
-        (structuresResults: Structure[]) => {
-          fetchStructuresActionCreator(structuresResults);
-        }
+      await supersetService(SUPERSET_PLANS_SLICE, supersetParams).then((result: Plan[]) =>
```

Comment:
If this is the last thing happening in `componentDidMount` then it doesn't need to `await` and we can drop `async` on the life-cycle method. 👍